themes/frontierline/functions.php

```diff
@@ -754,4 +754,23 @@ function save_frontierline_display_hero() {
 
 add_action('save_post', 'save_frontierline_display_hero');
 
+
+/**
+ * Use custom feed templates
+ */
+function frontierline_feed_rss2() {
+    load_template( STYLESHEETPATH . '/feeds/feed-rss2.php');
```

Comment:
It's to prevent the feed from revealing post authors. Removing the author name on the page template is easy enough but it's still a field in the standard feed, and at least for the Firefox blog they wanted to give the impression of a single user even though there are several people publishing. The press blogs also shouldn't have an author since several people publish there but all posts should appear to be from "Mozilla" as a collective.

We also discussed the idea of explicitly setting the author on every post (rather than leaving it to users to remember to change it themselves), which would allow us to use the standard feed template. And it just occurred to me while writing this comment that I wonder if there's already a plugin to do this...

Reply:
I moved the feeds folder into the parent theme. I don't know why github isn't showing it in the full  view but you can see the files being moved in the "jump to" menu.

Reply:
I see it now there, thanks. This way the author will be removed from the main theme feed too. Is that OK? If not I think guess it will be possible to add the templates depending on the preference, or add/remove the author from the feed based on the preference. This is just a minor comment IMO. For our use on Mozilla.cz this is not important and AFAIK it should be possible to deregister the templates again if someone wants to.

Reply:
We should only use the custom feeds if the no_byline option is true, otherwise it should use the standard feeds. I think we can probably just wrap this whole set of functions in a conditional.

Reply:
I added the author back to the RSS feeds wrapped in a conditional based on `no_byline`. This will allow us to make other changes  to the RSS templates if we want (I was thinking it might be good to add the featured images to them in the future.